.azure/pipelines/jobs/default-build.yml

```diff
@@ -73,11 +73,17 @@ jobs:
     ${{ if ne(parameters.poolName, '') }}:
       name: ${{ parameters.poolName }}
     ${{ if and(eq(parameters.poolName, ''), eq(parameters.agentOs, 'macOS')) }}:
-      name: Hosted macOS
+      ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+        name: Hosted Mac Internal
```

Comment:
Hosted macOS is fine for internal builds. Not sure what this queue offers, and it may not be worth switching.